File: self_c/ex0411/ex01.c
#include <stdio.h>

int main(){
    for(int i=0; i<10;i++)
        printf("Hello,world! %d\n",i);
        printf("for \n");
        
        int num1=1;
        switch(num1){
            case 1:
            printf("1임\n");
            //break;
            case 2:
            printf("2 또는 1임\n");
            break;
            default:
            printf("기본값임");
            break;
        }
}

File: 0507/ex01.js
const http=require("http");
//node에서만 사용 가능 브라우저에서 사용 불가

const server = http.createServer((req,res)=>{
    
    res.end('hello,world');
})

server.listen(8005,()=>{
    console.log('8005로 실행되었습니다.')
})